src/domain/login/LoginViewModel.ts

```diff
@@ -93,23 +110,23 @@ export class LoginViewModel extends ViewModel {
         this.emitChange("startSSOLoginViewModel");
     }
 
-    _showError(message) {
+    _showError(message: string) {
         this._errorMessage = message;
         this.emitChange("errorMessage");
     }
 
-    _setBusy(status) {
+    _setBusy(status: boolean) {
         this._isBusy = status;
         this._passwordLoginViewModel?.setBusy(status);
         this._startSSOLoginViewModel?.setBusy(status);
         this.emitChange("isBusy");
     }
 
-    async attemptLogin(loginMethod) {
+    async attemptLogin(loginMethod: PasswordLoginMethod) {
```

Comment:
Should this be `ILoginMethod` rather?

Reply:
Ah yes, makes sense. Double check me here -- I think [this](https://github.com/ibeckermayer/hydrogen-web/blob/f8c47059447cbc12962f12a6373b49cf634cb721/src/domain/login/LoginViewModel.ts#L112) is meant to be `TokenLoginMethod` and then [this](https://github.com/ibeckermayer/hydrogen-web/blob/f8c47059447cbc12962f12a6373b49cf634cb721/src/domain/login/LoginViewModel.ts#L126) is `PasswordLoginMethod`.